cvat-core/src/api-implementation.ts

```diff
@@ -33,39 +33,40 @@ import QualityConflict from './quality-conflict';
 import QualitySettings from './quality-settings';
 import { FramesMetaData } from './frames';
 import AnalyticsReport from './analytics-report';
+import CVATCore from '.';
 
-export default function implementAPI(cvat) {
-    cvat.plugins.list.implementation = PluginRegistry.list;
-    cvat.plugins.register.implementation = PluginRegistry.register.bind(cvat);
+function implementationMixin(func: Function, implementation: Function): void {
+    Object.assign(func, { implementation });
+}
+
+export default function implementAPI(cvat: CVATCore): CVATCore {
+    implementationMixin(cvat.plugins.list, PluginRegistry.list);
+    implementationMixin(cvat.plugins.register, PluginRegistry.register.bind(cvat));
 
-    cvat.lambda.list.implementation = lambdaManager.list.bind(lambdaManager);
-    cvat.lambda.run.implementation = lambdaManager.run.bind(lambdaManager);
-    cvat.lambda.call.implementation = lambdaManager.call.bind(lambdaManager);
-    cvat.lambda.cancel.implementation = lambdaManager.cancel.bind(lambdaManager);
-    cvat.lambda.listen.implementation = lambdaManager.listen.bind(lambdaManager);
-    cvat.lambda.requests.implementation = lambdaManager.requests.bind(lambdaManager);
+    implementationMixin(cvat.lambda.list, lambdaManager.list.bind(lambdaManager));
+    implementationMixin(cvat.lambda.run, lambdaManager.run.bind(lambdaManager));
+    implementationMixin(cvat.lambda.call, lambdaManager.call.bind(lambdaManager));
+    implementationMixin(cvat.lambda.cancel, lambdaManager.cancel.bind(lambdaManager));
+    implementationMixin(cvat.lambda.listen, lambdaManager.listen.bind(lambdaManager));
+    implementationMixin(cvat.lambda.requests, lambdaManager.requests.bind(lambdaManager));
 
-    cvat.server.about.implementation = async () => {
+    implementationMixin(cvat.server.about, async () => {
         const result = await serverProxy.server.about();
         return result;
-    };
-
-    cvat.server.share.implementation = async (directory: string, searchPrefix?: string) => {
+    });
+    implementationMixin(cvat.server.share, async (directory: string, searchPrefix?: string) => {
         const result = await serverProxy.server.share(directory, searchPrefix);
         return result.map((item) => ({ ...omit(item, 'mime_type'), mimeType: item.mime_type }));
-    };
-
-    cvat.server.formats.implementation = async () => {
+    });
+    implementationMixin(cvat.server.formats, async () => {
```

Comment:
Please, add newlines inbetween the implementations. Or maybe just between the groups cvat.`something`

Reply:
Right now there are newlines exactly between groups, like between cvat.server and cvat.assets, etc. 
If to add these spaces between each row, we will lose some kind of visual difference


We can't add 2 new blank lines, because eslint complaints about it

Reply:
Ok, I see